### ISSUE TITLE
Chime - Only buzzer is playing

### ISSUE THREAD

**Issue by jan-sevcik-snadnee** [hjdhjd/homebridge-unifi-protect#1144]
### Homebridge UniFi Protect Version

7.8.2

### Homebridge Platform and OS

Raspberry Pi - Raspbian

### Homebridge Version

1.8.5

### Node Version

20.15.0

### UniFi OS Version

4.0.20

### UniFi Protect Controller Version

 5.0.51

### Describe the problem

Hi, when I connect a Unifi Chime, I see it instantly in the Homekit with the ability to play one of 5 sounds. Unfortunately the only one working switch is "Buzzer". When I click to any other switch, nothing happens. See the log bellow.

### Homebridge HBUP JSON configuration

```json
{
    "controllers": [
        {
            "address": "XYZ",
            "username": "XYZ",
            "password": "XYZ"
        }
    ],
    "name": "UniFi Protect",
    "platform": "UniFi Protect",
    "_bridge": {
        "username": "0E:1B:DE:A8:D5:F3",
        "port": 59160
    },
    "options": [
        "Enable.Doorbell.Trigger.AC8BA96387D1",
        "Enable.Doorbell.PhysicalChime.AC8BA96387D1",
        "Enable.Doorbell.Volume.Dimmer.AC8BA96387D1",
        "Enable.Doorbell.PhysicalChime.Duration.Digital.AC8BA96387D1.1000",
        "Enable.Doorbell.Trigger.F4E2C60EAC7C"
    ]
}
```

### Relevant log output

```shell
[18/11/2024, 18:03:16] [UniFi Protect] Smart WiFi Chime [Smart WiFi Chime]: Playing buzzer.
[18/11/2024, 18:03:24] [UniFi Protect] Smart WiFi Chime [Smart WiFi Chime]: Unable to play Default.
[18/11/2024, 18:03:24] [UniFi Protect] Smart WiFi Chime [Smart WiFi Chime]: Playing Default.
[18/11/2024, 18:04:41] [UniFi Protect] Smart WiFi Chime [Smart WiFi Chime]: Unable to play Express-Line.
[18/11/2024, 18:04:41] [UniFi Protect] Smart WiFi Chime [Smart WiFi Chime]: Playing Express-Line.
[18/11/2024, 18:04:51] [UniFi Protect] Smart WiFi Chime [Smart WiFi Chime]: Unable to play Sundrops.
[18/11/2024, 18:04:51] [UniFi Protect] Smart WiFi Chime [Smart WiFi Chime]: Playing Sundrops.
[18/11/2024, 18:04:55] [UniFi Protect] Smart WiFi Chime [Smart WiFi Chime]: Unable to play Traditional.
[18/11/2024, 18:04:55] [UniFi Protect] Smart WiFi Chime [Smart WiFi Chime]: Playing Traditional.
```

### Acknowledgment that you are only running UniFi OS and UniFi Protect releases from the Ubiquiti Official release channel

- [X] I acknowledge that I am not running any early access / beta firmwares or operating systems from either Ubiquiti or Apple.

**Comment by hjdhjd** [comment]
Thanks for the report - I’ll look into this one over the weekend. Ubiquiti’s been introducing a lot of changes to chime support over the past few releases and keeping up can be a challenge. Thanks for raising this!

Will followup with any questions when I’m ready.

**Comment by jan-sevcik-snadnee** [comment]
Thank you for a super fast reply. Is there a change to also set the volume of the chime? Or any other settings? Maybe setting it in the plugin config would be enough for all of this, but its up to you! :)

**Comment by hjdhjd** [comment]
See above. I have nothing to add at this time. When I have more to say, I’ll add to this thread. Thanks. 😛 Please try to keep this thread clear. Speculation isn’t helpful to me helping you.

**Comment by jan-sevcik-snadnee** [comment]
IT was automatically closed, just adding a comment since this would be very useful to fix. Thank you! :)

**Comment by hjdhjd** [comment]
Alright, had some time for this over the weekend - so a couple of things:

1. I tested this issue out...chimes work fine when you aren't pairing them against a **physical** chime. All the capabilities work as they should. This is a positive sign leading to...
2. The chime 's capabilities are determined by your own setup and wiring. Specifically, it appears you've configured having a *physical* chime...I'm not sure if you're able to actually to trigger anything other than the buzzer. It's not a configuration I've used or tested much.

Leading to...what do you see when you look at the settings for your doorbell in the UniFi Protect webUI? Can you share a screenshot of what you see as your available settings? Are you able to play individual chime tones from the native Protect webUI? 

TL;DR: I'm not sure what you want to do is possible...but it may well be. Your specific setup (having a physical non-smart doorbell chime connected to a UniFi doorbell and additional WiFi chimes) is an outlier configuration that I haven't seen used in this context. Let's see if it's even possible. 😄

**Comment by jan-sevcik-snadnee** [comment]
Hi, yes you are right, I have a physical UniFi Smart Chime.

Since I don't have a UniFi doorbell yet, just the chime that I'm using standalone, I don't see any configuration in the UI - no able to play any sound. 

So this can be definitely the issue. The question is if it's possible to play any sound anyway and use it as an independent chime.

This is what I see in the Smart Chime settings. ![image](https://github.com/user-attachments/assets/ded19915-132f-4f79-8e00-21dde85443c7)

**Comment by hjdhjd** [comment]
I don't believe so. Given all that - closing this out more fully. Chimes are accessories for Protect doorbells, not independent devices in the Protect ecosystem.

Best of luck!
